pkg/snowflake/view.go

```diff
@@ -235,27 +234,3 @@ func ScanView(row *sqlx.Row) (*View, error) {
 	err := row.StructScan(r)
 	return r, err
 }
```

Comment:
We should be able to remove this whole file, right? or Drop and other operations are used somewhere?

Reply:
It was used in streams still, I though it was something bigger, but it seems I can get rid of that afterall.